client/app/components/about-modal/about-modal.html

```diff
@@ -1,7 +1,5 @@
-<a href="#"  ng-click="vm.open()" >
-    <span translate>
-      About
-    </span>
+<a href="#"  ng-click="vm.open()" translate>
```

Comment:
In this case, both variants are possible.

```html
<a translate>
...
</a>
```

wouldn't work only if the text of the `a` element would contain another html markup. In this case, it only contains the 'About' string, so it will work just fine.